docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -105,9 +105,14 @@ Findable
 		
 	- Authorship, contact information, and description:
 	
-		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through. 
+		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through.
+		
+	- Link GitHub repository:
+	
+		- Additionally, you can include a URL to your original GitHub repo README in order to connect the Dockstore entry with the source code. If your workflow is moved out of Dockstore, this will help connect it with the original source code. You may consider doing so especially when you have additional comprehensive README files available on GitHub.
```

Comment:
Are you saying add a link to the README from your descriptor file? If so, can you please specify that?

Also for both this section and the previous, workflows can have multiple descriptor files, so you could touch upon that (add to the primary?), but not crucial.

Reply:
@coverbeck I'm confused by your second statement, doesn't descriptor file = workflow? By "workflows can have multiple descriptor files", do you mean different versions of a descriptor file/workflow?

Reply:
I suggest "Additionally, for workflow languages that include meta sections, you can include a URL to your original GitHub repo README in the meta section of your descriptor file(s). If your workflow is moved out of Dockstore, this will help connect the descriptor file with the original source code and full documentation.

Reply:
> @coverbeck I'm confused by your second statement, doesn't descriptor file = workflow? By "workflows can have multiple descriptor files", do you mean different versions of a descriptor file/workflow?

One workflow = 1 to n descriptor files. For example, [this one](https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/assemble_refbased:master?tab=files) has many WDL files. Many (most?) workflows have just one descriptor, but many workflows have multiple descriptors.